utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/ViewMatcher.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.testing
+
+import android.content.res.Configuration
+import android.view.View
+import com.microsoft.device.dualscreen.utils.wm.DisplayPosition
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+/**
+ * Checks if view given its position {@see DisplayPosition} is on the display given its dimensions.
+ *
+ * @param position: target position.
+ * @param orientation: orientation of the display. See Configuration {@see Configuration}.
+ * @param firstDisplayWith : width of the left/start/first display area
+ * @param totalDisplayWith : width of the right/end/second/total display area
+ * @param foldingFeatureWidth : width of the FoldingFeature {@see FoldingFeature} if any.
+ * @return
+ */
+fun isViewOnScreen(
+    position: DisplayPosition,
+    orientation: Int,
+    firstDisplayWith: Int,
+    totalDisplayWith: Int,
```

Comment:
I think there are 2 typos here - `firstDisplayWith ` and `totalDisplayWith`. Should be Width, right?

Reply:
😨😱😱😱 totally!! thanks Bianca!